app/controllers/site_controller.rb

```diff
@@ -128,6 +130,17 @@ def update_study_settings
         set_study_permissions(@study.detached?)
         set_study_default_options
         set_study_download_options
+
+        # manage reviewer access
+        reviewer_opts = params.to_unsafe_hash['reviewer_opts']
```

Comment:
this hash seems less like an options hash, and more like actions.  so maybe rename to `reviewer_access_actions`?

Reply:
Fair point.

Reply:
446268b